app/domain/errors.rb

```diff
@@ -306,6 +306,16 @@ module AuthnK8s
         msg:  "Invalid Kubernetes host id: {0}. Must end with <namespace>/<resource_type>/<resource_id>",
         code: "CONJ00048E"
       )
+
+      CaResourceMissing = ::Util::TrackableErrorClass.new(
+        msg:  "Resource {0-resource-id} doesn't have saved resource for cert_id or key_id ",
```

Comment:
Do we have validation for log messages from Docs or is it not needed here?

Reply:
From where are we getting the `resource-id`?

Reply:
From the creation of the exception